.travis.yml

```diff
@@ -43,7 +52,7 @@ install:
         if [[ $DJANGO != 1.8* ]]; then
             pip install "django>=$DJANGO,<=$DJANGO.99";
         else
-            pip install https://www.djangoproject.com/download/$DJANGO/tarball/;
+            pip install Django==1.8c1 django-formtools;
```

Comment:
django 1.8 has been released.  Any reason this is using a release candidate?  Instead:

```
pip install Django==1.8 django-formtools;
```